.travis.yml

```diff
@@ -1,17 +1,13 @@
+---
```

Comment:
The .travis.yml file is now virtually the same as the one from the platform-services SDK.